GenderPayGap.WebUI/BackgroundJobs/ScheduledJobs/UpdatePublicFacingDownloadFilesJob.cs

```diff
@@ -94,6 +95,11 @@ private void SaveCsvFile(IEnumerable records, string relativeFilePath)
                 {
                     csvWriter.Configuration.ShouldQuote = (s, context) => true;
                     csvWriter.Configuration.TrimOptions = TrimOptions.InsideQuotes;
+
+                    var options = new TypeConverterOptions {Formats = new[] {"yyyy/MM/dd HH:mm:ss"}};

```

Comment:
This was the format the dates were using but I added this here just in case the default format is changed.